### PR TITLE
Add DTS enum to Interop ComCtl32

### PR DESCRIPTION
**Pull request by gpetrou**
## Proposed changes

- Add DTS enum to Interop ComCtl32.
- Remove DTS constants and replace their usages with the above enum values.
- Remove unused DI_NORMAL constant.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2893)

### CODE REVIEW

File: src/System.Windows.Forms.Primitives/src/Interop/ComCtl32/Interop.DTS.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class ComCtl32
+    {
+        [Flags]
+        public enum DTS : uint
```
Comment (JuditRose) [inline src/System.Windows.Forms.Primitives/src/Interop/ComCtl32/Interop.DTS.cs]: Why changing the type to uint from int?

File: src/System.Windows.Forms/src/System/Windows/Forms/DateTimePicker.cs [hunk 2]
```diff
@@ -935,11 +935,11 @@ public bool ShowCheckBox
         {
             get
             {
-                return (style & NativeMethods.DTS_SHOWNONE) != 0;
+                return (style & (int)DTS.SHOWNONE) != 0;
```
Comment (hughbe) [inline src/System.Windows.Forms/src/System/Windows/Forms/DateTimePicker.cs]: can we make `style` into a field of type `DTS`?

File: src/System.Windows.Forms/src/System/Windows/Forms/DateTimePicker.cs [hunk 3]
```diff
@@ -51,7 +51,7 @@ public class DateTimePicker : Control
 
         private static readonly string DateTimePickerLocalizedControlTypeString = SR.DateTimePickerLocalizedControlType;
 
-        private const int TIMEFORMAT_NOUPDOWN = NativeMethods.DTS_TIMEFORMAT & (~NativeMethods.DTS_UPDOWN);
+        private const int TIMEFORMAT_NOUPDOWN = (int)(DTS.TIMEFORMAT & (~DTS.UPDOWN));
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/DateTimePicker.cs]:
```suggestion
        private const DTS TIMEFORMAT_NOUPDOWN = DTS.TIMEFORMAT & ~DTS.UPDOWN;
```

File: src/System.Windows.Forms/src/System/Windows/Forms/DateTimePicker.cs [hunk 4]
```diff
@@ -902,43 +902,30 @@ public virtual bool RightToLeftLayout
...
         ///  value.
         /// </summary>
-        [
-        DefaultValue(false),
-        SRCategory(nameof(SR.CatAppearance)),
-        SRDescription(nameof(SR.DateTimePickerShowNoneDescr))
-        ]
+        [DefaultValue(false)]
+        [SRCategory(nameof(SR.CatAppearance))]
+        [SRDescription(nameof(SR.DateTimePickerShowNoneDescr))]
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/DateTimePicker.cs]: Please revert the changes to the decorations, @hughbe has https://github.com/dotnet/winforms/pull/2892 in flight.
